### PR TITLE
Gender input to radio selection, Adjust style client info page

### LINKED ISSUE #183: Convert gender question in the registration questionnaire from an input field to a radio button for gender selection
Implement the select gender question as a select radio button as opposed to an input text. With default selection male. See attached screenshots 

Current implementation 
Screen Shot 2021-05-31 at 12 12 31 PM

Correct mock 
Screen Shot 2021-05-31 at 12 12 31 PM

### PR DESCRIPTION
**Pull request by shoaibmushtaq25**
**IMPORTANT: Where possible all PRs must be linked to a Github issue**

Fixes #183 

**Type**
Choose one: Feature 

**Checklist**
- [x] I have written **Unit tests** for any new feature(s) and edge cases for bug fixes
- [x] I have updated the  [CHANGELOG.md](./CHANGELOG.md) file for any notable changes to the codebase
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the project's style guide
- [x] I have built and run the fhircore app to verify my change fixes the issue and/or does not break the app

### CODE REVIEW

**Comment by shoaibmushtaq25** [comment]
Screenshot of styling changes , also gender radio buttons
Screenshot_20210610-134832_FHIR Engine Example

**Review by ekigamba — CHANGES_REQUESTED** [review]

File: android/app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -22,7 +22,6 @@
         <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
         <item name="colorAccent">@color/colorAccent</item>
         <item name="android:listDivider">@color/white_slightly_opaque</item>
-        <item name="questionnaire_theme">@style/Theme.MaterialComponents.Light.DarkActionBar</item>
```
[the remark this answers is not in the document]
Comment (shoaibmushtaq25) [reply android/app/src/main/res/values/styles.xml:25]: Okay. Got it...I can revert this change but Let me explain my problem. We need to show top Title "Client info" as bold, so die to overriding of questionnaire theme, this top Title was not appearing as bold. Please suggest me the alternative to solve my problem? @ekigamba

File: android/app/src/main/res/drawable/rounded_white_bg.xml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
     <stroke android:color="#7AA2BD"
         android:width="0.5dp"/>
-    <corners android:radius="10dp" />
+    <corners android:radius="4dp" />
```
[the remark this answers is not in the document]
Comment (shoaibmushtaq25) [reply android/app/src/main/res/drawable/rounded_white_bg.xml:5]: For the drawable, I checked figma designs, there are different values of corner radius of different screens, i.e, 3 px, 6 px and 10 px. And we made it 10 px bit it looks off (corner radius too bigger). So I changed it to 4dp to make it consistant.. what do you think about it? @ekigamba

File: android/app/src/main/res/layout/activity_questionnaire.xml [hunk 3]
```diff
@@ -14,9 +14,40 @@
...
+        android:layout_marginRight="15dp"
+        android:layout_marginLeft="15dp"
+        android:background="@drawable/rounded_white_bg"
+        android:backgroundTint="@color/colorPrimary"
+        android:drawablePadding="5dp"
+        android:fontFamily="sans-serif-medium"
+        android:padding="10dp"
+        android:paddingHorizontal="10dp"
+        android:paddingEnd="15dp"
+        android:text="Save"
```
[the remark this answers is not in the document]
Comment (shoaibmushtaq25) [reply android/app/src/main/res/layout/activity_questionnaire.xml:46]: Yeah sure.

File: android/app/src/main/res/values/styles.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (ekigamba) [reply android/app/src/main/res/values/styles.xml:25]: Cool, kindly define a style that extends the theme above with the `groupHeaderTextAppearance` as in this file https://github.com/google/android-fhir/blob/master/datacapture/src/main/res/values/styles.xml#L36

File: android/app/src/main/res/drawable/rounded_white_bg.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (ekigamba) [reply android/app/src/main/res/drawable/rounded_white_bg.xml:5]: Cool this is OK

File: android/app/src/main/res/values/styles.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (shoaibmushtaq25) [reply android/app/src/main/res/values/styles.xml:25]: Thanks @ekigamba , I fixed it.

File: android/app/src/main/res/layout/activity_questionnaire.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (shoaibmushtaq25) [reply android/app/src/main/res/layout/activity_questionnaire.xml]: Fixed.

**Review by ndegwamartin — DISMISSED** [review]

**Comment by shoaibmushtaq25** [comment]
Latest screenshot after style change.
Screenshot_20210611-173150_FHIR Engine Example

**Review by ndegwamartin — APPROVED** [review]

**Pull request merged**